common/app/model/LiveBlogCurrentPage.scala

```diff
@@ -70,6 +74,18 @@ object LiveBlogCurrentPage {
         BlockPage(blocks = Nil, blockId = blockId, pageNumber = numPages, filterKeyEvents)
       }
 
+      // returns latest pinned post and blocks minus the pinned post
+      val (pinnedPost, filteredFirstPageBlocks) = {
+        val pinnedPosts = blocks.requestedBodyBlocks.get(CanonicalLiveBlog.pinned)
+        val pinnedPost = pinnedPosts.flatMap(_.headOption)
+        val pinnedPostId = pinnedPost.map(_.id).getOrElse("")
```

Comment:
instead of doing getorelse you can use the contains method on an option

Reply:
Yep updated